.github/workflows/create-release-main.yml

```diff
@@ -0,0 +1,71 @@
+name: Create new preview release (main)
```

Comment:
Suggestion: Would be better to make it sound like its pushing image per day/week? 

something like - Build and push nightly/weekly ASO image

Reply:
Let's figure out https://github.com/Azure/azure-service-operator/pull/3699#discussion_r1451794792 first and then we can align

Reply:
I think we need to be consistent with terminology - so far I've spotted _preview_, _experimental_ and _latest_ all used to describe this.

I suggest we standardize on _experimental_, as the other two terms have common meanings that don't suit (that is, latest = most recent release, preview = beta release).

I suggest
```
name: Create new release (experimental)
```

Reply:
Agreed, I will align things!

Reply:
Aligned